lib/go-atscfg/ipallowdotconfig.go

```diff
@@ -29,19 +29,80 @@ import (
 	"github.com/apache/trafficcontrol/lib/go-util"
 )
 
+// IPAllowConfigFileName is both the file name of ip_allow.config files and the
+// ConfigFile value of Parameters that can affect its contents.
 const IPAllowConfigFileName = `ip_allow.config`
+
+// ContentTypeIPAllowDotConfig is the MIME type of the contents of an
+// ip_allow.config file.
 const ContentTypeIPAllowDotConfig = ContentTypeTextASCII
+
+// LineCommentIPAllowDotConfig is the string that signifies the start of a line
+// comment in the grammar of an ip_allow.config file.
 const LineCommentIPAllowDotConfig = LineCommentHash
 
+// ParamPurgeAllowIP is the Name of Parameters which can specify an IP address
+// to be allowed ALL methods on the cache server (not just PURGE).
+//
+// TODO: replace instances of "purge_allow_ip" with this currently-unused
+// constant.
+// TODO: unexport? This is only used in this package.
```

Comment:
This shouldn't be removed, it exists for use outside ATC. Someone might want to write a script to manipulate these parameters, and using the canonical ATC `tc.ParamPurgeAllowIP` is the right way to do that.

Reply:
> ... using the canonical ATC `tc.ParamPurgeAllowIP` is the right way to do that.

but it's actually `atscfg.ParamPurgeAllowIP` - should the comment instead suggest moving it into `lib/go-tc`? Most constants declaring special Parameter Names and/or Config Files are in `lib/go-atscfg` from what I can see, but there's one in `lib/go-tc`.

I can see a case for asking them to be in either place, but it really may come down to where `lib/go-atscfg` will live if and when t3c moves to its own repository, IMO. I think the special Parameter definitions ought to remain with ATC, and t3c would just be in charge of implementing them. That's probably not something we should decide right now, though. TODOs don't even necessarily need to be TODONE.